features/bootstrap/FeatureContext.php

```diff
@@ -113,9 +122,10 @@ public function removeAcceptHeaderBeforeScenario()
      */
     public function createDatabase()
     {
-        $this->schemaTool->dropSchema($this->classes);
+        $this->isOrm() && $this->schemaTool->dropSchema($this->classes);
+        $this->isOdm() && $this->schemaManager->dropDatabases();
```

Comment:
could it make sense to split the context into two subclasses instead?

Reply:
There is a lot to separate at the end. I think we should move the complete extension stuff into separate classes as we did it in our `sonata-admin-integration-bundle` when the branches a really separated, we should see what each implementation is needing.